src/app/onion/learning-object-builder/components/content-upload/app/upload/upload.component.ts

```diff
@@ -229,32 +251,32 @@ export class UploadComponent implements OnInit, AfterViewInit, OnDestroy {
    * @private
    * @memberof ViewComponent
    */
-  private async fetchLearningObject() {
-    try {
-      this.retrieving = true;
-      this.learningObject = await this.learningObjectService.getLearningObject(
-        this.learningObjectName
-      );
-      this.config.url = USER_ROUTES.POST_FILE_TO_LEARNING_OBJECT(
-        this.learningObject.id
-      );
-      this.retrieving = false;
-      // FIXME: Add folder descriptions to entity
-      // ADD FOLDER DESCRIPTION PROP IF NOT EXIST
-      if (!this.learningObject.materials['folderDescriptions']) {
-        this.learningObject.materials['folderDescriptions'] = [];
-      }
-      this.updateFileSubscription();
-      this.updateFolderMeta();
-    } catch (e) {
-      this.notificationService.notify(
-        `Could not fetch Learning Object`,
-        `${e}`,
-        `bad`,
-        ``
-      );
-    }
-  }
+  // private async fetchLearningObject() {
```

Comment:
Is this going to be reintroduced at a later time? Why is it commented out right now?

Reply:
This is commented out right now because, since the content-upload application is no longer stand alone, it doesn't fetch the learning object directly. Rather, the object is injected through an input field from the builder-store service.